app/Keycloak/Dto/Realm.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak\Dto;
```

Comment:
I would even place this class and the `Config` class inside the `App\Keycloack` namespace. That was also done for all other domains likes Insightly, Auth0 and UiTiD.